app/controllers/users_controller.rb

```diff
@@ -16,6 +16,11 @@ def create
     end
   end
 
+  def show
+    user = User.find(params[:id])
+    @recommendables = user.recommendables
```

Comment:
I'd love to write that, but `user.recommendations` is the join model, not the actual TV show, book, etc.

> Introduce `Recommendation` join model, which polymorphically belongs to a recommendable. A recommendable is a television show, a book, a movie, etc.

I think `user.recommendations` reads MUCH better but I don't know what to call the join model if we name it that. In `has_many :recommendations, through: _____`, what should go in the blank?